components/ws-manager/pkg/manager/create.go

```diff
@@ -501,6 +501,15 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 		},
 	}
 
+	var terminationGracePeriodSeconds *int64
+	if req.Spec.TerminationGracePeriodSeconds != 0 {
+		terminationGracePeriodSeconds = &req.Spec.TerminationGracePeriodSeconds
+		var maxGracePeriodSeconds int64 = 60 * 5 // 5 minutes
```

Comment:
can we use `stopWorkspaceNormallyGracePeriod` const from manager.go here? and add a comment in manager.go for this const that it is used as a maximum for user supplied grace period.